EIPS/eip-7069.md

```diff
@@ -114,10 +116,13 @@ Before the 63/64th rule was introduced, it was required to calculate available g
 
 We have changed the ruleset:
 
-1. Removed the call depth check.
-2. Use the 63/64th rule, but
-    2a. ensure that at least 5000 gas is retained prior to executing the callee,
-    2b. ensure that at least 2300 gas is available to the callee.
+1. Use the 63/64th rule, but
+    - Ensure that at least `MIN_RETAINED_GAS` gas is retained prior to executing the callee,
+    - Ensure that at least `MIN_CALLEE_GAS` gas is available to the callee.
+
+Removing the call stack depth check was initially considered, but this would be incompatible with the original `*CALL` instructions, as well as `CREATE*` instructions, which can be intertwined with the new `EXT*CALL` instructions in the call stack. As such, keeping the call stack depth check involves no change affecting legacy code.
+
+Also, we find the simple (as opposed to the complex 63/64th rule) hard cap reassuring, that the call stack depth is limited, in case the gas rules can be bypassed. Lastly, the amount of gas to reach depth of 1024 is huge, but not absurdly huge and we want to avoid constrain ourselves by relying this check on current gas limits.
```

Comment:
```suggestion
Also, we find the simple (as opposed to the complex 63/64th rule) hard cap reassuring, that the call stack depth is limited, in case the gas rules can be bypassed. Lastly, the amount of gas to reach depth of 1024 is huge, but not absurdly huge, and we want to avoid constraining ourselves by dependency of this check on current gas limits.
```